src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -326,22 +356,33 @@ $pf-v6--include-toolbar-breakpoints: true !default;
       }
     }
 
-    // - Toolbar - Toolbar content - Toolbar content section
-    .#{$toolbar},
-    .#{$toolbar}__content,
-    .#{$toolbar}__content-section {
-      // Inset modifiers
-      @each $inset, $inset-value in $pf-v6-c-toolbar--inset-map {
-        &.pf-m-inset-#{$inset}#{$breakpoint-name} {
-          --#{$toolbar}__content--PaddingInline: #{$inset-value};
-        }
+    // - Toolbar inset
+    @each $inset, $inset-value in $pf-v6-c-toolbar--inset-map {
+      .#{$toolbar}.pf-m-inset-#{$inset}#{$breakpoint-name} {
+        --#{$toolbar}--PaddingInlineStart: #{$inset-value};
+        --#{$toolbar}--PaddingInlineEnd: #{$inset-value};
+      }
+    }
+
+    // - Toolbar content section
+    @each $inset, $inset-value in $pf-v6-c-toolbar--inset-map {
+      .#{$toolbar}__content.pf-m-inset-#{$inset}#{$breakpoint-name} {
+        --#{$toolbar}__content--PaddingInlineStart: #{$inset-value};
+        --#{$toolbar}__content--PaddingInlineStart: #{$inset-value};
+      }
+    }
+
+    // - Toolbar content section inset
+    @each $inset, $inset-value in $pf-v6-c-toolbar--inset-map {
+      .#{$toolbar}__content-section.pf-m-inset-#{$inset}#{$breakpoint-name} {
+        --#{$toolbar}__content-section--PaddingInlineStart: #{$inset-value};
+        --#{$toolbar}__content-section--PaddingInlineStart: #{$inset-value};
```

Comment:
For v5, `.pf-m-inset-*` looks to only apply to `.#{$toolbar}` - I'm curious the reason we need to support these modifiers on `__content` and `__content-section`?